var/spack/repos/builtin/packages/libgrpp/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Libgrpp(CMakePackage):
+    """A library for the evaluation of molecular integrals of the"""
+
+    """A library for the evaluation of molecular integrals of the generalized relativistic pseudopotential operator over Gaussian functions"""
+
+    homepage = "https://github.com/aoleynichenko/libgrpp"
+    git = "https://github.com/aoleynichenko/libgrpp.git"
+
+    maintainers("mtaillefumier")
+
+    version("master", branch="main")
```

Comment:
Why does the version name not match the branch?

We generally recommend that they be the same these days to avoid confusion between those on the host repository and listed in the package.  

You can see how versions arecompared in the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison.